Sources/Controllers/ALKConversationViewController.swift

```diff
@@ -2443,6 +2443,13 @@ extension ALKConversationViewController: ALMQTTConversationDelegate {
               UIApplication.shared.applicationState != .background else { return false }
         return true
     }
+    
+    public func handOff(toHuman conversationId: String!) {
```

Comment:
It should be like `handOffToHumanIn(conversationId: String)`. Also, remove `implicitly unwrapped optional` type.

Reply:
updated the name as `handOffToZendeskAgent`

Reply:
also allmqttdelegate methods are explicitly unwrapping when we implement in swift ..so added nil check before triggering the method